### PR TITLE
Nifty Close Price Prediction using Neural Networks

### LINKED ISSUE #234: 💡[FEATURE]: NIFY DATA ANALYSIS(PREDICTION OF CLOSE PRICE USING NEURAL NETWORKS)
**Is your feature request related to a problem? Please describe.**
There is no close price predictor of NIFTY. People invest their money in the wrong way by not being able to predict the current close price.

**Describe the solution you'd like**
I would develop a neural network that  would predict the close price of NIFTY.

**Describe alternatives you've considered**
-The feature would include proper data analysis of NIFTY
- Prediction of close price using the NIFTY dataset

### PR DESCRIPTION
**Pull request by Srijansarkar17**
The project is basically a Nifty Close Price Prediction model using Neural Networks and it has a loss of 3.4x10^-5.

## 🛠️ Related Issue

- Closes #234

## 📝 Description
The model is a Nifty Close Price prediction model which predicts the close price of the nifty. It is trained on neural networks. The activation function used is ReLU and the loss metric used for this project is mean_squared_logarithmic_error. The model has a loss of 3.4X10^-5 on the training set and 2.6x10^-5 on the test set. This project also called data visualization.

## 🔍 Type of PR

- [ ] 🐛 Bug fix
- [X] ✨ Feature enhancement
- [ ] 📚 Documentation update
- [ ] 🛑 Other (specify): _______________

## 📸 Screenshots / 🎥 Videos (if applicable)

[Attach screenshots or videos]
Screenshot 2024-06-03 212726
Screenshot 2024-06-03 212818
Screenshot 2024-06-03 212856
Screenshot 2024-06-03 212923

## ✅ Checklist

- [X] I have performed a self-review of my code.
- [X] I have read and followed the Contribution Guidelines.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Review by Akshat111111 — COMMENTED** [review]
Good work!, Kindly add your folder as Nifty Close Price Prediction inside Nifty folder then I will merge.Create a new PR for it
